README.md

```diff
@@ -25,55 +26,27 @@ The Busola also includes React and Angular libraries:
-
-2. Install Busola dependencies. To install dependencies for the root and all UI projects, and prepare symlinks for local libraries within this repository, run the following command:
+## Usage with NPX
 
-   ```bash
-   npm run bootstrap
-   ```
+This mode has no hot-reload feature so it's not designed for the development. Hovewer, it gives you the ability to run the Busola in a few seconds without installing any dependencies.
 
-   > **NOTE:** The `npm run bootstrap` command:
-   >
-   > - installs root dependencies provided in the [package.json](./package.json) file
-   > - installs dependencies for the [`React common`](./common), [`React components`](./components/react), [`Shared components`](./components/shared) and [`Generic documentation`](./components/generic-documentation) libraries
-   > - builds all the libraries
-   > - installs dependencies for all the [components](#components)
-   > - updates your `/etc/hosts` with the `127.0.0.1 busola-dev.kyma.local` host
-   > - creates the `.clusterConfig.gen` file if it doesn't exist, pointing at the `kyma.local` domain
+To run the Busola in the `npx` mode from the latest relase you have to provide `KUBECONFIG` environment variable pointing to a valid KUBECONFIG file, then type `npx @kyma-project/busola@latest` in your terminal. That's it, your default browser should open the Busola within a few seconds.
 
-## Usage
-
-### Set the cluster (optional)
+## Installation
 
-By default, the Kyma cluster URL with which the Busola communicates is set to `kyma.local`. To change the address of the cluster, run:
+To install dependencies for the root and all UI projects, and prepare symlinks for local libraries within this repository, run the following command:
 
 ```bash
-./scripts/.setClusterConfig {CLUSTER_URL}
+npm run bootstrap:ci
 ```
 
-To simplify switching clusters hosted on the same domain, you can assign the domain to `CLUSTER_HOST` environment variable, then use any subdomain as a cluster name.
-
-For example, let's assume you want to easily switch between two clusters - `foo.abc.com` and `bar.abc.com`. Follow these steps to simplify switching between these clusters:
-
-```bash
-export CLUSTER_HOST=abc.com
-# If you use only one domain for your cluster, consider setting it permanently in your shell.
-
-./scripts/.setClusterConfig foo
-# After setting the CLUSTER_HOST variable this is equal to running ./scripts/.setClusterConfig foo.abc.com
-
-./scripts/.setClusterConfig bar
-# Switch to a different cluster on the same domain
-```
+> **NOTE:** The `npm run bootstrap:ci` command:
+>
+> - installs root dependencies provided in the [package.json](./package.json) file
+> - installs dependencies for [the libraries](###Components)
```

Comment:
This link is broken. It should be probably #components, but the TW will know more. Or you can see the governance job to check if the links are broken or not :D